client/src/client.rs

```diff
@@ -251,9 +252,9 @@ where
 #[display(fmt = "{}@{torii_url}", "key_pair.public_key()")]
 pub struct Client {
     /// Url for accessing iroha node
-    torii_url: Url,
+    torii_url: SocketAddr,
```

Comment:
I would also rename it to `torii_addr`, but it is a breaking change for the configuration. Although it might be deprecated, but we don't have a process for this for now.

Reply:
@appetrosyan `SocketAddr` can be `host:port` too.
The reason I left it as `Url` here is Torii could also be behind a reverse proxy that puts it under `host:port/path`, which is, well, an `Url`

Reply:
Which means this **needs** to be reverted.

Reply:
it has already been reverted